contracts/staking/tests/PoSValidatorManagerTests.t.sol

```diff
@@ -557,25 +601,36 @@ abstract contract PoSValidatorManagerTest is ValidatorManagerTest {
         _setUpCompleteEndDelegation(validationID, delegationID1, DEFAULT_WEIGHT, DEFAULT_WEIGHT, 4);
     }
 
-    function testCompleteEndValidation() public override {
+    function testCompleteEndValidation() public virtual override {
+        uint256 registrationDuration = 1000 * 60 * 60 * 24; // 1 day
+
+        uint256 registrationExpiry = vm.getBlockTimestamp() + registrationDuration;
+
         bytes32 validationID = _setUpInitializeEndValidation({
             nodeID: DEFAULT_NODE_ID,
             subnetID: DEFAULT_SUBNET_ID,
             weight: DEFAULT_WEIGHT,
-            registrationExpiry: DEFAULT_EXPIRY,
+            registrationExpiry: uint64(registrationExpiry),
             blsPublicKey: DEFAULT_BLS_PUBLIC_KEY,
             registrationTimestamp: DEFAULT_REGISTRATION_TIMESTAMP,
-            completionTimestamp: DEFAULT_COMPLETION_TIMESTAMP
+            completionTimestamp: uint64(registrationExpiry),
+            includeUptime: true
         });
 
         uint256 balanceBefore = _getStakeAssetBalance(address(this));
 
+        uint256 expectedReward = DEFAULT_WEIGHT * DEFAULT_REWARD_RATE / 365 - 1;
```

Comment:
expected rewards shouldn't be based on weight, but on the value staked which is currently a 12 order of magnitude difference